spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/actions/BaseSparkAction.java

```diff
@@ -200,6 +203,18 @@ protected Dataset<FileInfo> manifestListDS(Table table, Set<Long> snapshotIds) {
     return toFileInfoDS(manifestLists, MANIFEST_LIST);
   }
 
+  protected Dataset<FileInfo> statisticsFileDS(Table table, Set<Long> snapshotIds) {
+    Predicate<StatisticsFile> predicate;
+    if (snapshotIds == null) {
+      predicate = statisticsFile -> true;
+    } else {
+      predicate = statisticsFile -> snapshotIds.contains(statisticsFile.snapshotId());
+    }
+
+    return toFileInfoDS(
+        ReachableFileUtil.statisticsFilesLocations(table, predicate), STATISTICS_FILES);
```

Comment:
nit: Can we define an extra var with a relatively short name like `statisticsFiles` or `locations` so that this would fit on one line?